e2e-tests/package.json

```diff
@@ -8,9 +8,14 @@
     "dotenv": "^8.2.0",
     "near-api-js": "^0.36.3",
     "near-seed-phrase": "^0.1.0",
-    "playwright": "^1.7.1"
+    "playwright": "^1.13.1"
   },
   "scripts": {
-    "test": "node -r dotenv/config index.js"
+    "test": "node -r dotenv/config index.js",
+    "e2e": "npx playwright test"
+  },
+  "devDependencies": {
```

Comment:
1. You need to use `yarn add` to add these so that the `yarn.lock` file is updated
2. These should actually be primary `dependencies`, not `devDependencies`, because these packages are actually part of the primary use case for this package -- to test the wallet end-to-end :). In a standard package where these dependencies only exist to be used when running internal tests against the code in `e2e-tests`, then they'd be dev dependencies...but that's not the case here :)

Reply:
Done 👍